nnpdf_data/nnpdf_data/new_commondata/LHCB_Z0J_13TEV_2022/kinematics_dimuon_pT.yaml

```diff
@@ -0,0 +1,169 @@
+bins:
+- pT:
+    min: 0.0
+    mid: 1.1
+    max: 2.2
```

Comment:
@enocera I'm a bit worried about the kinematics here. For sure no fixed order calculation will be reliable at pT ~ 0, but I'm not even sure pT ~ 10 will be ok...

Reply:
Indeed. The cut on pT that we have for the ATLAS and CMS 8 TeV data sets is at 30 GeV, therefore anything lower than that cannot be described by FO pert QCD (but needs resummation). So are you saying that the maximum pT in that measurement is 10 GeV? if this is the case, then we may forget about it until we do pT resummation.

Reply:
Looking at the paper we may have, say, 3 bins above pT~30 GeV. Anyways, at this point I'd complete the implementation of the data set - that may be useful for the future.